src/app/tasks/task-list.model.ts

```diff
@@ -0,0 +1,9 @@
+import { Task } from "./task.model";
+
+export interface TaskList {
+    tasks: Task[]
+}
+
+export let emptyTaskList: TaskList = {
```

Comment:
I think it should be const, no? Since you will not be changing this object anywhere